components/brave_vpn/resources/panel/components/toggle/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import * as S from './style'
+
+interface Props {
+  on: boolean
```

Comment:
Would `isOn` be more easily understandable considering we have `isX` patterns in most places (`isActive` for `Box`)? Plus, `on` is very _React_ for events, so the prefix could help readability.

Reply:
this makes sense